packages/react-native-gesture-handler/src/v3/types.ts

```diff
@@ -106,6 +106,8 @@ export type SingleGesture<THandlerData, TConfig> = {
   gestureRelations: GestureRelations;
 };
 
+export type SingleGestureType = SingleGesture<unknown, unknown>;
```

Comment:
What do you think about a union: `TapGestureType | PanGestureType | ...`

It would have to be defined elsewhere, though.